WPMedia/fusion-news-theme-blocks
# THEMES-1066: Update Storybook to v8

## Description

#### Jira Ticket: [THEMES-1066](https://arcpublishing.atlassian.net/browse/THEMES-1066)

This PR updates Storybook to the latest version and makes the necessary updates to our config so the stories render.

## Test Steps

1. Checkout this branch `git checkout THEMES-1066`
2. Run `npm i` and then `npm run storybook`
3. Ensure that stories are loading in the browser and that the styling looks correct. Note that only the `news.scss` is being loaded, so commerce blocks may be missing their unique styling.


[THEMES-1066]: https://arcpublishing.atlassian.net/browse/THEMES-1066?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
